src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -32,5 +32,9 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    public String getFeedback() {
+    	return feedbackToUser;
+    }
```

Comment:
Did you compile the program after this change? TextUi.java should have changed too.